### PR TITLE
BUG 2076290: Update Readme : How to configure T-BC/OC and enable events

### PR DESCRIPTION
**Pull request by aneeshkp**
Update upstream documentation with details on how PTPconfig and
PTPOperator are used to enable PTP events and also to show how to configure Bndary ock (T-BC)
and Ordinary clock ptpConfig

Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,9 +36,11 @@ metadata:
   resourceVersion: ""
   selfLink: ""
 ```
-
-Below is an example of updating `daemonNodeSelector` to select all worker nodes:
-
+### Enable PTP events via fast event framework 
+PTP Operator supports fast event publisher for events such as PTP state change, os clock out of sync, clock class change and  port failure.
```
Comment (josephdrichard) [inline README.md:40]: extra space

File: README.md [hunk 2]
```diff
@@ -55,24 +57,37 @@ items:
...
+linuxptp configurations shall be applied. 
+The Spec of CR has two major sections. 
+The first section `profile` contains `interface`, `ptp4lOpts` , `phc2sysOpts`  and `ptp4lConf` options,
+the second `recommend` defines profile selection logic.
+```
+ PTP operator supports T-BC and Ordinary clock which can be configured via ptpConfig
+```
+### ptpConfig to set up ordinary clock using single interface
+``` 
+NOTE: following ptp4l/phc2sys opts required when  events are enabled
```
Comment (josephdrichard) [inline README.md:82]: extra space

File: README.md (hunk 1 above)
Comment (aneeshkp) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (aneeshkp) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -55,24 +57,37 @@ items:
...
-
-
 ## PtpConfig
 
-`PtpConfig` CRD is used to define linuxptp configurations and to which node these linuxptp configurations shall be applied. The Spec of CR has two major sections. The first section `profile` contains `interface`, `ptp4lOpts` and `phc2sysOpts` options, the second `recommend` defines profile selection logic.
-
+`PtpConfig` CRD is used to define linuxptp configurations and to which node these 
+linuxptp configurations shall be applied. 
+The Spec of CR has two major sections. 
+The first section `profile` contains `interface`, `ptp4lOpts` , `phc2sysOpts`  and `ptp4lConf` options,
```
Comment (josephdrichard) [inline README.md:75]: extra space here.

Comment (aneeshkp) [reply README.md]: Thanks , ran the tool to check extra space and I believe everything is  fixed now

**Pull request merged**